### PR TITLE
RM-8856 Add missing integration test in LocalizedValidationMessagesGlobalizationIntegrationTest

### PR DESCRIPTION
**Pull request by KuJo7**
https://re-motion.atlassian.net/browse/RM-8856

### CODE REVIEW

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 1]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          4);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@"2.*4"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:444]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@" 2 .* 4 "));
```
if this results in a green test, too.

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 2]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          1);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@"2.*4"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:475]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@" 2 .* 4 "));
```
If this results in green test, too.

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 3]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          "test");
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@" 10 "));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:211]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain(" 10 "));
```

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 4]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          1);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain("2"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:267]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain(" 2 "));
```

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 5]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          2);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain("3"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:295]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain(" 3 "));
```

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 6]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          3);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain("2"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:323]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain(" 2 "));
```

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 7]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          2);
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain("1"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:351]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain(" 1 "));
```

File: Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs [hunk 8]
```diff
@@ -121,10 +128,352 @@ public void CultureIsSetAfterValidationResultIsCreated_UsesPreviousCultureForLoc
...
+          instanceToValidate,
+          propertyStub,
+          "test");
+
+      using (new CultureScope(culture))
+      {
+        var validationFailures = validator.Validate(context).ToArray();
+
+        Assert.That(validationFailures.Length, Is.EqualTo(1));
+        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Match(@" 10 ?"));
```
Comment (MichaelKetting) [inline Remotion/Validation/IntegrationTests/LocalizedValidationMessagesGlobalizationIntegrationTest.cs:183]:
```suggestion
        Assert.That(validationFailures[0].LocalizedValidationMessage, Does.Contain(" 10 "));
```

**Review by MichaelKetting — APPROVED** [review]
Please verify, I have added some whitespace and replaced regex with simple contains. So long as the resoruce texts use whitespace left and right of the numbers, all should still work. if not, we can still remove the whitespace padding.

**Comment by KuJo7** [comment]
@MichaelKetting pls check one last time before I merge, I needed to remove the space after the number for some tests, because there is a `.` instead of a space

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**